model_compression_toolkit/gptq/common/gptq_config.py

```diff
@@ -19,9 +19,12 @@
 
 class RoundingType(Enum):
     """
-    An enum for choosing the GPTQ rounding methods
+    An enum for choosing the GPTQ rounding methods:
+
     0. STRAIGHT-THROUGH ESTIMATOR
```

Comment:
regarding previous comment - this needs to be modified to reflect the correct enum values (names) in the generated documentation)